### ISSUE TITLE
How to extract/save the loss function

### ISSUE THREAD

**Issue by isalirezag** [junyanz/pytorch-CycleGAN-and-pix2pix#94]
Sorry for my stupid question.
I am wondering in the training and testing process, is there any way to extract the loss values for each input and save them somewhere?

**Comment by junyanz** [comment]
You can call [get_current_errors](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/8bc6778456c86a8b9d88362efb195635d2a0dac3/train.py#L31) to get the losses.

**Comment by isalirezag** [comment]
@junyanz How about in testing mode?

**Comment by junyanz** [comment]
You can call the same function in the test mode.

**Comment by isalirezag** [comment]
@junyanz Sorry but I am a little confused.
Can we even have Loss for the testing procedure? If so what loss would it be? because I do not see any loss in the testing codes...

**Comment by junyanz** [comment]
It's possible. But it is not implemented in the current repo. You can copy and paste the `backward_` functions, and comment `loss_*.backward()` in the each function.
